articles/active-directory/fundamentals/scenario-azure-first-sap-identity-integration.md

```diff
@@ -260,7 +260,7 @@ Consider building automation to execute the entire certificate rollover process.
 
 ## Using Azure AD B2C as the Identity Provider
 
-[Azure Active Directory B2C](../../active-directory-b2c/overview.md) provides business-to-customer identity as a service. Given that the integration with Azure AD B2C is similar to how you would allow enterprise users to sign in with Azure AD, the recommendations above still mostly apply when you want to use Azure AD B2C for your customers, consumers or citizens and allow them to use their preferred social, enterprise, or local account identities. There are a few important differences, however.
+[Azure Active Directory B2C](../../active-directory-b2c/overview.md) provides business-to-customer identity as a service. Given that the integration with Azure AD B2C is similar to how you would allow enterprise users to sign in with Azure AD, the recommendations above still mostly apply when you want to use Azure AD B2C for your customers, consumers or citizens and allow them to use their preferred social, enterprise, or local account identities. Setting up Azure AD B2C as a corporate identity provider in IAS and configuring federation between both tenants is described in more detail in [this blog post](https://blogs.sap.com/2023/02/08/identity-federation-between-azure-ad-b2c-and-sap-cloud-identity-services-using-custom-policies/). There are a few important differences, however.
```

Comment:
The closing sentence of this block is now disconnected from the first argument. Please revise. One option could be to move the injected sentence into a new block below.

Reply:
Agree; the flow can be better.

Reply:
I've re-ordered the last paragraph and added a line break for better clarity.